auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -214,16 +214,22 @@ private void showRegisterEmailLinkFragment(AuthUI.IdpConfig emailConfig,
     }
 
 
-    private void switchFragment(Fragment fragment, String tag, boolean withTransition) {
+    private void switchFragment(Fragment fragment, String tag, boolean withTransition,
```

Comment:
Can we make an overload without the new param that defaults it to false?

Reply:
We have one already that sets both booleans to false.